addons/website_sale/static/src/js/website_sale.js

```diff
@@ -7,18 +7,15 @@ import { extraMenuUpdateCallbacks } from "@website/js/content/menu";
 import "@website/libs/zoomodoo/zoomodoo";
 import { ProductImageViewer } from "@website_sale/js/components/website_sale_image_viewer";
 import VariantMixin from "@website_sale/js/sale_variant_mixin";
-import { cartHandlerMixin } from "@website_sale/js/website_sale_utils";
 
 
-export const WebsiteSale = publicWidget.Widget.extend(VariantMixin, cartHandlerMixin, {
+export const WebsiteSale = publicWidget.Widget.extend(VariantMixin, {
     selector: '.oe_website_sale',
     events: Object.assign({}, VariantMixin.events || {}, {
         'change form .js_product:first input[name="add_qty"]': '_onChangeAddQuantity',
-        'click a.js_add_cart_json': '_onClickAddCartJSON',
+        'click a.js_add_cart_json': '_setQuantity',
```

Comment:
Confusing method name IMHO, `setQuantity` would be more for the above event, this one is only decreasing/incrasing the quantity if I understand correctly.

Reply:
`_onClickAddCartJSON` is way more confusing IMO, what name would you suggest then?

Reply:
change/modifyQuantity ?
set is more the action of manually specifying a value, which wouldn't exactly cover the event's scope